README.md

```diff
@@ -1,21 +1,28 @@
-# OpenAI Gym wrapper for the DeepMind Control Suite.
+# OpenAI Gym wrapper for the DeepMind Control Suite
+
 A lightweight wrapper around the DeepMind Control Suite that provides the standard OpenAI Gym interface. The wrapper allows to specify the following:
+
 * Reliable random seed initialization that will ensure deterministic behaviour.
 * Setting ```from_pixels=True``` converts proprioceptive observations into image-based. In additional, you can choose the image dimensions, by setting ```height``` and ```width```.
 * Action space normalization bound each action's coordinate into the ```[-1, 1]``` range.
 * Setting ```frame_skip``` argument lets to perform action repeat.
 
+## Installation
 
-### Instalation
-```
-pip install git+git://github.com/denisyarats/dmc2gym.git
+```shell
+pip install git+git://github.com/kmdanielduan/dmc2gym.git
 ```
 
-### Usage
+## Usage
+
 ```python
-import dmc2gym
+import gym
+from dm_control import suite
+from dmc2gym import register_suite, register_env, dmc_task2str

```

Comment:
Is `register_env` used anywhere in the example?

Reply:
No, it's not used in the examples. I guess I could remove it.